Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=569378
Author: Emilio Pozuelo Monfort <pochu@debian.org>

Move the define before using it to fix a build failure.

diff -ruNp gnet-2.0.8/tests/check/gnetcheck.h gnet-2.0.8.new/tests/check/gnetcheck.h
--- gnet-2.0.8/tests/check/gnetcheck.h	2008-02-03 13:04:41.000000000 +0100
+++ gnet-2.0.8.new/tests/check/gnetcheck.h	2010-03-06 19:23:28.000000000 +0100
@@ -310,6 +310,8 @@ int main (int argc, char **argv)				\
 
 gboolean _gnet_check_run_test_func (const gchar * func_name);
 
+#define _tcase_add_test  __gnet_tcase_add_test
+
 static inline void
 __gnet_tcase_add_test (TCase * tc, TFun tf, const gchar * func_name,
     int sig, int start, int end)
@@ -319,8 +321,6 @@ __gnet_tcase_add_test (TCase * tc, TFun
   }
 }
 
-#define _tcase_add_test  __gnet_tcase_add_test
-
 G_END_DECLS
 
 #endif /* __GNET_CHECK_H__ */
